external/bugswarm/scripts/download_artifact.py

```
@@ -0,0 +1,62 @@
+    )
+    args = parser.parse_args()
+
+    artifact_id = args.artifact_id.strip()
+    if not artifact_id:
+        print("Empty --artifact-id", file=sys.stderr)
+        return 1
+    repo = (args.image_repo or os.environ.get("BUGSWARM_IMAGE_REPO") or DEFAULT_IMAGE_REPO).strip()
+    image = f"{repo}:{artifact_id}"
+
+    try:
+        subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)